### PR TITLE
OVS-4304 - Remove vpool throws exception

### PR DESCRIPTION
**Pull request by kvanhijf**
Initial commit was just removal of test level functionality
Same can be attained by listing the tests and executing specific set of tests (autotests.run([test1, test2, ...])

WIP

### CODE REVIEW

**Comment by kvanhijf** [comment]
In [1]: from ci import autotests

In [2]: autotests.run('ci.tests.sanity')
nose.config: INFO: Set working dir to /opt/OpenvStorage/ci/tests
nose.config: INFO: Working directory /opt/OpenvStorage/ci/tests is a package; adding to sys.path
nose.config: INFO: Ignoring files matching ['^\\.', '^_', '^setup\\.py$']
ci.tests.sanity.sanity_checks_test.TestSanity.config_files_check_test ... ok
ci.tests.sanity.sanity_checks_test.TestSanity.json_files_check_test ... ok
ci.tests.sanity.sanity_checks_test.TestSanity.services_check_test ... ok
ci.tests.sanity.sanity_checks_test.TestSanity.ssh_check_test ... ok
ci.tests.sanity.sanity_checks_test.TestSanity.system_services_check_test ... ok

----------------------------------------------------------------------
Ran 5 tests in 0.242s

OK

File: ci/tests/general/general_alba.py [hunk 1]
```diff
@@ -223,13 +232,204 @@ def add_alba_backend(name, wait=True):
...
+        :param alba_backend: ALBA backend
+        :return: None
+        """
+        # Validate presets
+        alba_backend.invalidate_dynamics('presets')
+        preset_available = False
+        for preset in alba_backend.presets:
+            if 'policy_metadata' not in preset:
+                raise ValueError('Preset information does not contain policy_metadata')
+            if preset_available is True:
```
Comment (pploegaert) [inline ci/tests/general/general_alba.py]: move to the end of the loop

File: ci/tests/general/general_alba.py [hunk 2]
```diff
@@ -251,45 +451,97 @@ def remove_alba_backend(alba_backend):
...
+        :return: Namespace information
+        """
+        nss = GeneralAlba.execute_alba_cli_action(alba_backend, 'list-namespaces')
+        if len(nss) > 0:
+            actual_keys = nss[0].keys()  # Only check 1st namespace for keys, assume rest is identical
+            expected_keys = ['id', 'name', 'nsm_host_id', 'preset_name', 'state']
+            assert set(actual_keys) == set(expected_keys), 'Expected keys did not match actual keys in namespace {0}'.format(nss[0])
+
+        # Apply filter
+        if kwargs.get('name') is not None:
```
Comment (pploegaert) [inline ci/tests/general/general_alba.py]: as you're using **kwargs might be interesting to filter on other values too io name only ...

Comment (kvanhijf) [inline ci/tests/general/general_alba.py]: The kwargs was implemented to allow more filtermethods, but at the time of implementation i only required the 'name' filter

**Pull request merged**